model/patch/patch.go

```diff
@@ -143,6 +143,8 @@ type Patch struct {
 	// tasks/variants are now scheduled to run). If true, the patch has been
 	// finalized.
 	Activated bool `bson:"activated"`
+	// Branch contains the branch that the project tracks.
+	Branch string `bson:"branch_name" json:"branch_name,omitempty"`
```

Comment:
Since we have both Branch and Project, can we clarify that the `Project` field has the `branch` BSON tag for legacy reasons, and that `Project` is really the project ID?

Reply:
oh sure.